### PR TITLE
frontend: profile add link on the username

### PR DESCRIPTION
**Pull request by t-h2o**
# frontend: profile add link on the username

add a link to `/user/${username}` on the name of the user

## Resources

Code [example of custom link](https://github.com/remix-run/react-router/blob/dev/examples/custom-link/src/App.tsx)

I started with this
[use link props](https://reactnavigation.org/docs/use-link-props/#options)
but we use react router and not
reactnavigation

## TODO

* [x] wait on #77
* [x] wait on #79

## Try

Click on the username on a friend in the friendlist

### CODE REVIEW

File: frontend/src/components/profile/Friend.tsx [hunk 1]
```diff
@@ -4,6 +4,34 @@ import IconRemoveFriend from '../../assets/icon/remove_friend.svg'
 import IconGenericPicture from '../../assets/icon/generic_picture.svg'
 import ClickableIcon from './ClickableIcon'
 
+import {
+    Routes,
+    Route,
+    Outlet,
```
Comment (eprei) [inline frontend/src/components/profile/Friend.tsx:10]: Do we need to import these three elements?

Comment (t-h2o) [reply frontend/src/components/profile/Friend.tsx:10]: Yes, sure, if I execute `make build`,
I receive:
```
src/components/profile/Friend.tsx(8,5): error TS6133: 'Routes' is declared but its value is never read.
src/components/profile/Friend.tsx(9,5): error TS6133: 'Route' is declared but its value is never read.
src/components/profile/Friend.tsx(10,5): error TS6133: 'Outlet' is declared but its value is never read.
```

**Review by eprei — APPROVED** [review]
Nice !

**Review by fire-poy — APPROVED** [review]

**Review by shaolin-peanut — APPROVED** [review]
very nice

**Pull request merged**